ssp

```diff
@@ -164,19 +164,14 @@ sub init {
                     $apache_built = $1;
                     $apache_built =~ s/^\s+//g;
                 }
-                if ( !$ea4 ) {
-                    if (m{ \A Cpanel::Easy::Apache \s+ (.*) \z }xms) {
-                        $apache_ea_version = $1;
-                    }
+                if (m{ \A Cpanel::Easy::Apache \s+ (.*) \z }xms) {
+                    $apache_ea_version = $1;
                 }
             }
-
-            if ($ea4) {
-                chomp( $apache_ea_version = timed_run( 0, 'rpm', '-qf', $httpd_bin ) );
```

Comment:
Although this is expensive it does provide useful information regarding the EA4 installation.  The installed EA4 RPM version is the modern equivalent of the old EA3 version and this is informative.  I do not believe this should be removed.  Leveraging @RPM_LIST or @EXTENDED_RPM_LIST may be the solution but those typically weren't available until later in the SSP run.

Reply:
What are the specific circumstances where this information *has* to be obtained as soon as an analyst logs in? Removing this check accounts for about half of the speed increase. Is there another way this use case can be covered?

Reply:
This argument could be made for any of the information that SSP provides.  I'm not currently aware of a way to obtain this information for EA4 without calling rpm because unfortunately this version information doesn't appear to be baked into the 'httpd' binary like it was with EA3.  To avoid a regression in functionality can we keep this one in until we figure out a better way to provide the same info?